src/main/java/redlightBack/Board/BoardService.java

```diff
@@ -18,19 +19,22 @@ public class BoardService {
     private final PostQueryRepository postQueryRepository;
     private final MemberRepository memberRepository;
 
-    //게시판 생성
-    public BoardResponse create (String userId, CreateBoardRequest request){
-        Board board = new Board(request.boardName());
+    // 게시판 생성 (관리자만)
+    public BoardResponse create(String userId, CreateBoardRequest request) {
 
-        memberRepository.findByUserId(userId).orElseThrow(
-                () -> new AccessDeniedException("접근 권한이 없습니다.")
-        );
+        // ADMINISTRATOR 권한이 아니면 예외
+        if (!memberRepository.existsByUserIdAndRole(userId, Role.ADMINISTRATOR)) {
```

Comment:
아래처럼 추상화하면 더 좋을 것 같아요.
```suggestion
        if (!hasAdministratorAccess(userId)) {
```